### PR TITLE
centered a lot of stuff, resized text, reorganized how information wa…

### PR DESCRIPTION
**Pull request by wle8300**
…s presented

### CODE REVIEW

File: about-us.html [hunk 1]
```diff
@@ -12,6 +12,22 @@
...
+    <link rel="prefetch" href="./assets/images/avatars/sam.png">
+    <link rel="prefetch" href="./assets/images/avatars/nic.png">
+    <link rel="prefetch" href="./assets/images/avatars/harvey.png">
+    <link rel="prefetch" href="./assets/images/avatars/logan.png">
+    <link rel="prefetch" href="./assets/images/avatars/wade.png">
+    <link rel="prefetch" href="./assets/images/avatars/toby.png">
+    <link rel="prefetch" href="./assets/images/right key.svg">
+    <link rel="prefetch" href="./assets/images/left key.svg">
+    <link rel="prefetch" href="./assets/images/right key pressed.svg">
+    <link rel="prefetch" href="./assets/images/left key pressed.svg">
```
Comment (ckcollab) [inline about-us.html:29]: tyvm!

File: about-us.html [hunk 2]
```diff
@@ -102,24 +113,13 @@
                                      :key="i">
                         <v-sheet height="100%">
                             <v-row class="description-row">
-                                <v-col class="description-sizer" cols="12" md="6">
+                                <v-col class="description-sizer" cols="12" md="6" style="width: 515px; max-width: 515px;">
```
Comment (ckcollab) [inline about-us.html]: should we avoid inline styling on new stuff?

**Comment by ckcollab** [comment]
Ty LGTM code wise!

**Comment by gibsonbailey** [comment]
@wle8300 Tightening up of content toward the center of the page looks great imho. What was your rationale for moving the blue `Contact Us` button to the top of the screen?

**Comment by wle8300** [comment]
@gibsonbailey i think to emphasize that the "Contact Us" button is the main call-to-action.

it kinda communicates to prospective clients they "hey, just reach out and we'll talk"

i still think it needs some slight tweaking, like maybe make the text **bold** and add schnazzy hover/press effects

**Pull request merged**